server/repository/src/db_diesel/invoice_line_row.rs

```diff
@@ -121,6 +121,18 @@ impl<'a> InvoiceLineRowRepository<'a> {
         Ok(())
     }
 
+    pub fn update_inventory_adjustment_reason_id(
```

Comment:
Added for separate update reason step as discussed in the PR description, rather than as part of insert_stock_in_line

Reply:
thanks for the comment